test/topology_custom/test_replace_with_encryption.py

```diff
@@ -7,42 +7,54 @@
 from test.pylib.manager_client import ManagerClient
```

Comment:
> This PR increases coverage for existing code. Hence we need to backport it. Since only 6.2 version has zero
token node support, hence we only backport it to 6.2

AFAIK, we don't backport new or extended tests, but I also recall cases when we considered a test important and backported it anyway, so I'm not sure.

Reply:
Backporting the test won't hurt if it's not too much of a hurdle. It shouldn't be in this case, it should backport automatically cleanly